app/src/main/java/com/example/tilsocial/DashboardActivity.java

```diff
@@ -47,23 +47,23 @@ public boolean onNavigationItemSelected( MenuItem menuItem) {
                     actionBar.setTitle("Home");
                     HomeFragment fragment = new HomeFragment();
                     FragmentTransaction fragmentTransaction = getSupportFragmentManager().beginTransaction();
-                    fragmentTransaction.replace(R.id.your_placeholder, fragment, "");
```

Comment:
@manpreet1719 fragment transaction code is being repeated multiple times. Pl create a Util class and move this method there and make it common